src/components/Popups/ReplyField/ReplyForm.tsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import Button from 'box-ui-elements/es/components/button';
+import PrimaryButton from 'box-ui-elements/es/components/primary-button';
+import { EditorState } from 'draft-js';
+import { FormikProps } from 'formik';
+import isEmpty from 'lodash/isEmpty';
+import { FormattedMessage, useIntl } from 'react-intl';
+import ReplyField from './ReplyFieldContainer';
+import { FormValues, Props as OtherProps } from './ReplyFormContainer';
+import messages from '../messages';
+
+type Props = OtherProps & FormikProps<FormValues>;
+
+const ReplyForm = ({
+    errors,
+    handleSubmit,
+    isPending,
+    onCancel,
+    onChange,
+    setFieldValue,
+    values,
+}: Props): JSX.Element => {
+    const intl = useIntl();
+    const hasErrors = !isEmpty(errors);
+    const handleChange = (editorState: EditorState): void => {
+        const text = editorState.getCurrentContent().getPlainText();
+        onChange(text);
```

Comment:
We should delegate this to Formik to avoid having to manage it ourselves. There should be a callback provided.

Reply:
There is, but the trick to getting Formik to work with DraftJS apparently, is to use the underlying `setFieldValue` method to set the entire `editorState` as the value. This is mainly to keep the current behavior where the `message` gets set back to the redux store. 

An alternative approach would be to not keep the `message` in store, this could just become:
```
<ReplyField
    ...
    onChange={setFieldValue}
    ...
/>
```
But then the downside is that the message wouldn't be available on subsequent attempts to redraw the annotation

Reply:
Zoom in/out will also clear the message

Reply:
Isn't Formik's biggest draw to keep the messages in the redux store?

Reply:
They kind of address that here: https://jaredpalmer.com/formik/docs/overview#why-not-redux-form. 
> According to our prophet Dan Abramov, form state is inherently ephemeral and local, so tracking it in Redux (or any kind of Flux library) is unnecessary

Reply:
That's true in most cases, but our components can be destroyed at any time and outside of our control. The redux store is persistent, though, so it should be used for anything that can't be quickly recovered, such as user-entered text.